src/features/surveys/components/SurveySuborgsCard.tsx

```diff
@@ -0,0 +1,48 @@
+import { ChangeEvent } from 'react';
+import messageIds from '../l10n/messageIds';
+import SurveyDataModel from '../models/SurveyDataModel';
+import { useMessages } from 'core/i18n';
+import useModel from 'core/useModel';
+import ZUICard from 'zui/ZUICard';
+
+import { Box, Switch } from '@mui/material';
+
+const SurveySuborgsCard = ({
+  orgId,
+  surveyId,
+}: {
+  orgId: number;
+  surveyId: number;
+}) => {
+  const messages = useMessages(messageIds);
+  const model = useModel((env) => new SurveyDataModel(env, orgId, surveyId));
+  const { data } = model.getData();
+  const orgAccess = data?.org_access === 'sameorg' ? false : true;
+
+  const handleChange = (ev: ChangeEvent<HTMLInputElement>) => {
+    if (ev.target.checked) {
+      model.updateSurveyAccess('suborgs');
+    } else {
+      model.updateSurveyAccess('sameorg');
+    }
+  };
+
+  return (
+    <Box paddingTop={2}>
+      <ZUICard
+        header={messages.shareSuborgsCard.title()}
+        status={
+          <Switch
+            checked={orgAccess}
+            onChange={(ev) => handleChange(ev)}
+          ></Switch>
+        }
+        subheader={messages.shareSuborgsCard.caption()}
+      >
+        {}
+      </ZUICard>
```

Comment:
If this card is empty, just use an empty tag (`<ZUICard … />` instead of `<ZUICard …>…</ZUICard>`.

Reply:
The card is empty but Typescript complains about that the children property missing. I'll need to add this to the card         children={undefined}

Reply:
Okay, Lint doesn't like that solution, it complains:  `error  Do not pass children as props. Instead, nest children between the opening and closing tags  react/no-children-prop`

Reply:
How about if you change the props type of `ZUICard` to say `children?: ReactNode` (instead of just `children: ReactNode`)?